app/Jobs/ProcessPendingWebsites.php

```diff
@@ -99,14 +99,16 @@ public function handle(): void
                         $publicAdministration = $website->publicAdministration;
 
                         if ($publicAdministration->status->is(PublicAdministrationStatus::PENDING)) {
-                            $pendingUser = $publicAdministration->users()->where('status', UserStatus::PENDING)->first();
-                            if (null !== $pendingUser) {
-                                $pendingUser->publicAdministrations()->detach($publicAdministration->id);
+                            $pendingUser = $publicAdministration->users()->where('user_status', UserStatus::PENDING)->first();
+                            $pendingUser->publicAdministrations()->detach($publicAdministration->id);
+
+                            if ($pendingUser->publicAdministrations->isEmpty()) {
                                 $pendingUser->deleteAnalyticsServiceAccount();
-                                $publicAdministration->forceDelete();
                             }
-                            $userEmailForPublicAdministration = $this->getUserEmailForPublicAdministration($pendingUser, $publicAdministration);
-                            event(new PublicAdministrationPurged($publicAdministration->toJson(), $pendingUser, $userEmailForPublicAdministration));
```

Comment:
@pdavide Non va bene passare l'indirizzo email dell'utente in questo punto? Dopo mi sembra che la pubblica amministrazione venga cancellata e quindi non sia più possibile fare la query per averlo.
Trovi che ci siano delle criticità?

Reply:
Pensavo di usare l’indirizzo che abbiamo già nella tabella users, ma adesso che me lo fai notare mi sembra più coerente inviare la notifica alla mail specifica per la pa.